tests/integration/components/public/ticket-list-test.js

```diff
@@ -1,14 +1,49 @@
 import { test } from 'ember-qunit';
+import Ember from 'ember';
 import moduleForComponent from 'open-event-frontend/tests/helpers/component-helper';
 import hbs from 'htmlbars-inline-precompile';
 
-moduleForComponent('public/ticket-list', 'Integration | Component | public/ticket list');
+const { Object: EmberObject, A } = Ember;
 
-const tickets = [{ description: 'Community ticket', date: 'Mon, May 22', price: 2.50, name: 'Community Ticket', type: 'paid', id: 1, quantity: 10, orderQuantity: 0, min: 0, max: 5 },
-  { description: 'Standard ticket', date: 'Mon, May 22', price: 3.05, name: 'Standard Ticket', type: 'paid', id: 2, quantity: 10, orderQuantity: 0, min: 0, max: 5 },
-  { description: 'Super ticket', date: 'Mon, May 22', price: 5.00, name: 'Super Ticket', type: 'paid', id: 3, quantity: 10, orderQuantity: 0, min: 0, max: 5 }
-];
+moduleForComponent('public/ticket-list', 'Integration | Component | public/ticket list');
 
+const tickets =  A(
+  [
+    EmberObject.create({
```

Comment:
Instead of ember object, could you create instances of the ticket model itself ?